src/games/tictactoe/board.tsx

```diff
@@ -47,6 +47,10 @@ export class Board extends React.Component<IBoardProps, {}> {
     return (this.props.gameArgs && this.props.gameArgs.mode === GameMode.OnlineFriend);
   }
 
+  isOfflineGame() {
```

Comment:
This check is happening in GameOver.tsx now, you dont need this. This is also the reason the coverage is decreasing. After you delete this line it should be good to merge

Reply:
Thank you!